app/views/krikri/field_value_reports/show.csv.erb

```diff
@@ -0,0 +1 @@
+<%= @field_value_report.headers.to_csv %><% @field_value_report.enumerate_rows.each do |row| %><%= row.to_csv %><% end %>
```

Comment:
This has no line breaks b/c they show up in the CSV file as blank lines.  Sorry, I know it's not nice to read.